src/bottom-bar/data-item-bar.js

```diff
@@ -18,6 +18,7 @@ export default function DataItemBar() {
             </span>
 
             <Button
+                secondary
                 small
                 className={styles.dataDetailsButton}
```

Comment:
can we remove the className, now that there is no `dataDetailsButton` class? Otherwise, looks good to me

Reply:
Yes, definitely. Done. Nice spot 👍